lib/features/offline_mode/manager/detailed_email_cache_manager.dart

```diff
@@ -25,6 +27,9 @@ class DetailedEmailCacheManager {
     if (listDetailedEmails.length >= CachingConstants.maxNumberNewEmailsForOffline) {
       final latestEmail = listDetailedEmails.last;
       log('DetailedEmailCacheManager::handleStoreDetailedEmail():latestEmail: $latestEmail');
+      if (latestEmail.emailContentPath != null) {
+        await _deleteFileExisted(latestEmail.emailContentPath!);
```

Comment:
- Currently we are saving the detailed email of 2 cases `OpenedEmail` and `NotificationEmail` into the same folder as `folderPath` with the same file name by id in internal storage. So this deletion will affect these 2 cases because it has different `maxNumber`.
- IMO, we should create 2 folders `opened_email` and `new_email` in `folderPath`